tests/test_deployment.py

```diff
@@ -1,10 +1,10 @@
-import os
-import pytest
 import glob
 import json
+import os
```

Comment:
I am surprised the import is really missing?

Reply:
No it's not, its just the IDE sorting the imports.